Use of autoLogin Plugin and run-workers leads to DOMException [DataCloneError]

Sorry, I currently don't have the time to write all details. It looks like a bug. Please have a look at the error message.

```
CodeceptJS v2.3.1
Running tests in 2 workers...

internal/worker.js:144
    this[kPort].postMessage({
                ^

DOMException [DataCloneError]: I => fetchFunc(I) could not be cloned.
    at new Worker (internal/worker.js:144:17)
    at /workdir/node_modules/codeceptjs/lib/command/run-workers.js:71:14
    at Array.map (<anonymous>)
    at createWorkers (/workdir/node_modules/codeceptjs/lib/command/run-workers.js:69:28)
    at /workdir/node_modules/codeceptjs/lib/command/run-workers.js:63:24
    at module.exports (/workdir/node_modules/codeceptjs/lib/hooks.js:27:13)
    at Command.module.exports (/workdir/node_modules/codeceptjs/lib/command/run-workers.js:62:3)
    at Command.listener (/workdir/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:209:13)
    at Command.parseArgs (/workdir/node_modules/commander/index.js:651:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
```

--- a/lib/command/run-workers.js
+++ b/lib/command/run-workers.js
@@ -60,17 +60,15 @@ module.exports = function (workers, options) {
 
   // run bootstrap all
   runHook(config.bootstrapAll, () => {
-    const workerList = createWorkers(groups, config, options, testRoot);
+    const workerList = createWorkers(groups, options, testRoot);
     workerList.forEach(worker => assignWorkerMethods(worker, groups.length));
   }, 'bootstrapAll');
 
-  function createWorkers(groups, config, options, testRoot) {
-    delete config.mocha;
+  function createWorkers(groups, options, testRoot) {
     const workers = groups.map((tests, workerIndex) => {
       workerIndex++;
       return new Worker(pathToWorker, {
         workerData: {
-          config: simplifyObject(config, { objects: false, underscores: false }),
           options: simplifyObject(options),
           tests,
           testRoot,
@@ -159,7 +157,7 @@ function appendStats(newStats) {
 }
 
 function repackTest(test) {
-  test = Object.assign(new Test(test.title || '', () => {}), test);
+  test = Object.assign(new Test(test.title || '', () => { }), test);
   test.parent = Object.assign(new Suite(test.parent.title), test.parent);
   return test;
 }
@@ -169,22 +167,13 @@ function repackSuite(suite) {
 }
 
 
-function simplifyObject(object, remove = {}) {
-  const defaultRemove = {
-    objects: true,
-    functions: true,
-    underscores: true,
-  };
-
-  remove = Object.assign(defaultRemove, remove);
-
-  let tempObj = Object.keys(object);
-  if (remove.underscores) tempObj = tempObj.filter(k => k.indexOf('_') !== 0);
-  if (remove.functions) tempObj = tempObj.filter(k => typeof object[k] !== 'function');
-  if (remove.objects) tempObj = tempObj.filter(k => typeof object[k] !== 'object');
-
-  return tempObj.reduce((obj, key) => {
-    obj[key] = object[key];
-    return obj;
-  }, {});
+function simplifyObject(object) {
+  return Object.keys(object)
+    .filter(k => k.indexOf('_') !== 0)
+    .filter(k => typeof object[k] !== 'function')
+    .filter(k => typeof object[k] !== 'object')
+    .reduce((obj, key) => {
+      obj[key] = object[key];
+      return obj;
+    }, {});
 }

